tools/coverage/Makefile

```diff
@@ -5,9 +5,20 @@ REGISTRY ?= gcr.io
 PROJECT  ?= knative-tests
 PUSH     ?= docker push
 
-coverage-image:
+# Making dev version of coverage-image
+coverage-image-dev:
+	@echo "Making dev version..."
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o coverage
 	chmod +x coverage
 	docker build -t "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:$(TAG)" -t "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:latest-dev" .
 	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:$(TAG)"
-	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:latest-dev"
\ No newline at end of file
+	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:latest-dev"
+
+# Making production version of coverage-image
+production:
+	@echo "Making production version..."
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o coverage
+	chmod +x coverage
+	docker build -t "$(REGISTRY)/$(PROJECT)/test-infra/coverage:$(TAG)" -t "$(REGISTRY)/$(PROJECT)/test-infra/coverage:latest" .
+	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage:$(TAG)"
+	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage:latest"
```

Comment:
in light of recent Knative OMG, should we also separate dev vs. prod image for coverage? 
any script is deleting those images, pls make sure that we don't repeat similar pattern of a script wiping out both dev and prod images.